app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Http\Requests\StoreCategoryRequest;
+use App\Http\Requests\UpdateCategoryRequest;
+use App\Services\CategoryService;
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    protected $categoryService;
+
+    // Constructor to inject CategoryService
+    public function __construct(CategoryService $categoryService)
+    {
+        $this->categoryService = $categoryService;
+    }
+
+    // Get all categories
+    public function index()
+    {
+        $categories = $this->categoryService->getAllCategories();
+        return response()->json($categories);
```

Comment:
Please define a resource class to return the data. This ensures a consistent and structured format for API responses. Also, ensure that the index results are paginated to handle large datasets efficiently. Each endpoint in the application should have a unified response structure to maintain consistency and improve the API's readability and usability.